.github/workflows/tox.yml

```diff
@@ -3,6 +3,7 @@ name: tox
 on:
   create:  # is used for publishing to PyPI and TestPyPI
     tags:  # any tag regardless of its name, no branches
+      - "*"
```

Comment:
AFAIK a null entry has the same effect at `["**"]` for patterns. I am not sure if single `*` also works but even if it does it should make no difference to what we have configured now.

Reply:
I didn't notice it was a double asterisk, I'll update that in my PR.

Reply:
I tried to mimic what https://github.com/ansible-community/molecule-docker/commit/60a4a9444726c029866cde6810f96a6c4a1fd4c4 did, which I thought fixed PyPi uploads for molecule-docker.